cvat-core/src/server-proxy.js

```diff
@@ -569,7 +571,8 @@
                             if (response.status === 202) {
                                 setTimeout(request, 3000);
                             } else {
-                                url = `${url}&action=download`;
+                                query = `${query}&action=download`;
```

Comment:
Suggest to use some (library)[https://www.npmjs.com/package/qs], to work with query as object

Reply:
@nglazov 

Looks like overhead, to add library for such a minor case
What is more, similar functionality for working with query string is already part of native JS: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

Reply:
Yes, it's good solution too
I mean, that better to work with objects instead of strings

Reply:
It makes sense